<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>DialogCreate</class>
  <widget class="QDialog" name="dialog_create">
    <property name="windowTitle">
      <string>Create Graphene</string>
    </property>
    <property name="modal">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
              <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                  <widget class="QLabel" name="label">
                    <property name="text">
                      <string>Width (Å):</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="spin_width">
                    <property name="minimum">
                      <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                      <double>1000.000000000000000</double>
                    </property>
                    <property name="value">
                      <double>50.000000000000000</double>
                    </property>
                    <property name="decimals">
                      <number>2</number>
                    </property>
                    <property name="singleStep">
                      <double>0.100000000000000</double>
                    </property>
                  </widget>
                </item>
                <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                    <property name="text">
                      <string>Height (Å):</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="spin_height">
                    <property name="minimum">
                      <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                      <double>1000.000000000000000</double>
                    </property>
                    <property name="value">
                      <double>50.000000000000000</double>
                    </property>
                    <property name="decimals">
                      <number>2</number>
                    </property>
                    <property name="singleStep">
                      <double>0.100000000000000</double>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                  <widget class="QLabel" name="label_3">
                    <property name="text">
                      <string>Center at:</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <layout class="QGridLayout" name="gridLayout_2">
                    <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_x">
                        <item row="0" column="0">
                          <widget class="QLabel" name="label_4">
                            <property name="text">
                              <string>x:</string>
                            </property>
                          </widget>
                        </item>
                        <item row="0" column="1">
                          <widget class="QDoubleSpinBox" name="spin_center_x">
                            <property name="minimum">
                              <double>0.000000000000000</double>
                            </property>
                            <property name="maximum">
                              <double>1000.000000000000000</double>
                            </property>
                            <property name="value">
                              <double>0.000000000000000</double>
                            </property>
                            <property name="decimals">
                              <number>2</number>
                            </property>
                            <property name="singleStep">
                              <double>1.000000000000000</double>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                    <item row="0" column="1">
                      <layout class="QGridLayout" name="gridLayout_y">
                        <item row="0" column="0">
                          <widget class="QLabel" name="label_5">
                            <property name="text">
                              <string>y:</string>
                            </property>
                          </widget>
                        </item>
                        <item row="0" column="1">
                          <widget class="QDoubleSpinBox" name="spin_center_y">
                            <property name="minimum">
                              <double>0.000000000000000</double>
                            </property>
                            <property name="maximum">
                              <double>1000.000000000000000</double>
                            </property>
                            <property name="value">
                              <double>0.000000000000000</double>
                            </property>
                            <property name="decimals">
                              <number>2</number>
                            </property>
                            <property name="singleStep">
                              <double>1.000000000000000</double>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                    <item row="0" column="2">
                      <layout class="QGridLayout" name="gridLayout_z">
                        <item row="0" column="0">
                          <widget class="QLabel" name="label_6">
                            <property name="text">
                              <string>z:</string>
                            </property>
                          </widget>
                        </item>
                        <item row="0" column="1">
                          <widget class="QDoubleSpinBox" name="spin_center_z">
                            <property name="minimum">
                              <double>0.000000000000000</double>
                            </property>
                            <property name="maximum">
                              <double>1000.000000000000000</double>
                            </property>
                            <property name="value">
                              <double>0.000000000000000</double>
                            </property>
                            <property name="decimals">
                              <number>2</number>
                            </property>
                            <property name="singleStep">
                              <double>1.000000000000000</double>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                    <item row="1" column="0">
                      <widget class="QCheckBox" name="check_pbc_x">
                        <property name="text">
                          <string>Periodic Boundary: x</string>
                        </property>
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="maximumSize">
                          <size>
                            <width>170</width>
                            <height>30</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="1">
                      <widget class="QCheckBox" name="check_pbc_y">
                        <property name="text">
                          <string>Periodic Boundary: y</string>
                        </property>
                        <property name="sizePolicy">
                          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                          </sizepolicy>
                        </property>
                        <property name="maximumSize">
                          <size>
                            <width>170</width>
                            <height>30</height>
                          </size>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <widget class="QLabel" name="label_9">
                    <property name="text">
                      <string>- If you want to create a CNT, let those options off -</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                      <widget class="QLabel" name="label_7">
                        <property name="text">
                          <string>Scale factor:</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QDoubleSpinBox" name="spin_scale">
                        <property name="minimum">
                          <double>1.000000000000000</double>
                        </property>
                        <property name="maximum">
                          <double>1000.000000000000000</double>
                        </property>
                        <property name="value">
                          <double>100.000000000000000</double>
                        </property>
                        <property name="decimals">
                          <number>2</number>
                        </property>
                        <property name="singleStep">
                          <double>10.000000000000000</double>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QLabel" name="label_8">
                        <property name="text">
                          <string>%</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QDialogButtonBox" name="buttonBox">
          <property name="orientation">
            <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
          <property name="centerButtons">
            <bool>false</bool>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections>
    <connection>
      <sender>buttonBox</sender>
      <signal>accepted()</signal>
      <receiver>dialog_create</receiver>
      <slot>accept()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>248</x>
          <y>254</y>
        </hint>
        <hint type="destinationlabel">
          <x>157</x>
          <y>274</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>buttonBox</sender>
      <signal>rejected()</signal>
      <receiver>dialog_create</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>316</x>
          <y>260</y>
        </hint>
        <hint type="destinationlabel">
          <x>286</x>
          <y>274</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
